src/components/Modals/OnboardingTransfer.vue

```diff
@@ -492,41 +719,64 @@ export default Vue.extend({
           arrowColor: '',
           placeholderColor: placeholderText,
           placeholder: placeholderText,
-          acronym: ''
+          acronym: '',
         },
         score: {
           shown: false,
           stopClose: false,
           boxBorder: '',
           arrowColor: '',
           placeholderColor: '',
-          placeholder: placeholderText,
-          acronym: ''
-        }
+          placeholder: '0',
+          acronym: '',
+        },
       };
       this.displayOptions.exam.push(exam);
+      this.setSubjectList();
+      this.getCredits();
     },
     getCourseFromExam(type: 'AP' | 'IB', subject: string) {
       let courses: Record<string, number> | undefined;
       for (const exam of reqsData[type]) {
         if (exam.name === subject) {
-          courses = exam.fulfillment.courseEquivalents
+          courses = exam.fulfillment.courseEquivalents;
           // as a default takes the first equivalent course
           // TODO will need to add requirements menu if editiable.
```

Comment:
Do you know what this is referring to?

Reply:
That's when it looks through the dummy JSON Ayesha made to find an equivalent course for an AP/IB test.